### ISSUE TITLE
AttributeError: module 'openai' has no attribute 'error'

### ISSUE THREAD

**Issue by bpanahij** [anarchy-ai/LLM-VM#390]
```
Traceback (most recent call last):
  File "/llmvm_testing/test.py", line 2, in <module>
    from llm_vm.client import Client
  File "/LLM-VM/src/llm_vm/client.py", line 6, in <module>
    from llm_vm.completion.optimize import LocalOptimizer
  File "/LLM-VM/src/llm_vm/completion/optimize.py", line 16, in <module>
    from llm_vm.completion.data_synthesis import DataSynthesis
  File "/LLM-VM/src/llm_vm/completion/data_synthesis.py", line 13, in <module>
    class DataSynthesis:
  File "/LLM-VM/src/llm_vm/completion/data_synthesis.py", line 57, in DataSynthesis
    @backoff.on_exception(backoff.expo, openai.error.RateLimitError)
```

It appears a recent (last few days) change to OpenAI's api is causing an error in the code after first downloading a model configs:

src/llm_vm/completion/data_synthesis.py: line 57

```
 @backoff.on_exception(backoff.expo, openai.error.RateLimitError)
    def generate_examples(self, final_prompt, openai_key, example_delim="<END>", model="gpt-4", max_tokens=1000, temperature=1, completion=None, call_big_kwargs={}):
```

The latest openai api removes the "error" intermediary, and references ```openai.RateLimitError``` like so.

**Comment by VictorOdede** [comment]
Hey @bpanahij thanks for highlighting this error. Seems to be an issue with the `backoff` lib. Will look into it asap.

**Comment by kshitiz305** [comment]
@bpanahij were you running data_synthesis.py

**Comment by kshitiz305** [comment]
Got it you are running test.py

**Comment by bpanahij** [comment]
It's a relatively small fix to correct the reference to RateLimitError. It of course doesn't address the underlying backoff issue.

**Comment by VictorOdede** [comment]
Closed by #392
